.pre-commit-config.yaml

```diff
@@ -1,72 +1,72 @@
-repos:
```

Comment:
Please remove the changes to this file (from Unix line endings to Windows line endings). Probably something your editor or git did unintentionally.

Reply:
Fixed!

Pre-commit threw an error and recommended to add the YAML to the commit. This was an unforseen consequence. Different clients do not always show when the line endings of a file changed or at least not as explicit as is shown here.